python/lsst/ts/logging_and_reporting/web_app/main.py

```diff
@@ -133,6 +134,34 @@ async def read_exposures(
         raise HTTPException(status_code=500, detail=str(e))
 
 
+@app.get("/expected-exposures")
+async def read_expected_exposures(
+    request: Request,
+    dayObsStart: int,
+    dayObsEnd: int,
+    # auth_token: str = Depends(get_access_token),
+):
+    logger.info(f"Getting expected exposures for start: {dayObsStart}, end: {dayObsEnd} ")
+    try:
+        expected_exposures = get_expected_exposures(
+            dayObsStart,
+            dayObsEnd,
+            # auth_token=auth_token
+        )
+
+        return {
+            "nightly_exposures": expected_exposures["nightly"],
+            "sum_exposures": expected_exposures["sum"],
+        }
+
+    except BaseLogrepError as ble:
+        logger.error(f"Rubin-sim error in /expected-exposures: {ble}")
+        raise HTTPException(status_code=502, detail="Rubin-sim API query failed")
```

Comment:
Is this exception really going to be triggered? I don't see any reference to it in the `scheduler_service.py`.